### PR TITLE
Fixing act errors in terminal

### PR DESCRIPTION
**Pull request by srmukher**
Before:
image

After:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d9b31e39451be52f4a1a2ab295bb20aec29275d9'>d9b31e39451be52f4a1a2ab295bb20aec29275d9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/utilities/testData.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { IVSChartDataPoint, IVerticalStackedChartProps } from '../index';
```
Comment (AtishayMsft) [inline packages/react-charting/src/utilities/testData.ts:2]: rename to test-data.test.ts and include in npmignore

Comment (srmukher) [reply packages/react-charting/src/utilities/testData.ts:2]: Are we not maintaining camel casing in case of file names?

File: packages/react-charting/.npmignore [hunk 2]
```diff
@@ -37,3 +37,4 @@ visualtests
 !lib-amd
 !dist
 docs
+testData.test.ts
```
Comment (AtishayMsft) [inline packages/react-charting/.npmignore:40]: I see *.test.* in npmignore. This line is not needed then.

File: packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxisRTL.test.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,1001 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`HorizontalBarChartWithAxis - mouse events Should render callout correctly on mouseover 1`] = `
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxisRTL.test.tsx.snap:3]: is the content for the 2 snapshots same?

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx [hunk 4]
```diff
@@ -569,7 +569,7 @@ describe('Line chart - Subcomponent xAxis Labels', () => {
     { data: dateChartPoints, rotateXAxisLables: true },
     container => {
       // Assert
-      expect(getByClass(container, /tick/i)[0].getAttribute('transform')).toContain('translate(40.5,0)');
+      expect(getByClass(container, /tick/i)[0].getAttribute('transform')).toContain('translate(55.07565789473684,0)');
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:572]: what is causing this change?

File: packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxisRTL.test.tsx.snap (hunk 3 above)
Comment (srmukher) [reply packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxisRTL.test.tsx.snap:3]: nope, snap for test and RTL test are different

File: packages/react-charting/src/utilities/testData.ts (hunk 1 above)
Comment (AtishayMsft) [reply packages/react-charting/src/utilities/testData.ts]: using kebab casing is the norm for react typescript and followed across fluent. Not sure why we are following camel casing. We can start with kebab casing for the tests at least.

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx (hunk 4 above)
Comment (srmukher) [reply packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:572]: no obvious change, after taking pull from master, I got this snapshot mismatch.

File: packages/react-charting/jest.config.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ const config = createConfig({
   moduleNameMapper: {
     ...getEsmOnlyPackagesToCjsMapping(),
   },
+  modulePathIgnorePatterns: ['<rootDir>/src/utilities/testData.test.ts'],
```
Comment (AtishayMsft) [inline packages/react-charting/jest.config.js:26]: why this change

File: packages/react-charting/.npmignore [hunk 6]
```diff
@@ -36,4 +36,4 @@ visualtests
 !lib-commonjs
 !lib-amd
 !dist
-docs
```
Comment (AtishayMsft) [inline packages/react-charting/.npmignore:39]: what is the change here

File: packages/react-charting/jest.config.js (hunk 5 above)
Comment (srmukher) [reply packages/react-charting/jest.config.js:26]: because otherwise running the tests also expects a test to be present within the test file

Comment (AtishayMsft) [reply packages/react-charting/jest.config.js:26]: then remove the test keyword and add entry in npmignore

File: packages/react-charting/.npmignore (hunk 6 above)
Comment (srmukher) [reply packages/react-charting/.npmignore:39]: added test-data.ts

File: change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30143&commitId=e2606ee39d1061517722b2d977bb729b02baba37&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+  "type": "none",
+  "comment": "Fixing act errors in terminal",
```
Comment (Hotell) [inline change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json:3]:
```suggestion
  "comment": "test: fix act errors in terminal",
```

File: packages/react-charting/jest.config.js (hunk 5 above)
Comment (Hotell) [reply packages/react-charting/jest.config.js:26]: I used following pattern https://github.com/microsoft/fluentui/pull/30117/files#diff-5e34a7bb2b3ed4a1697b714cd896f3587ab5f081495dd722712d658f6a4c55ab

this will also mitigate need of change files (as nothing related to implementation is changed)

File: change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json (hunk 8 above)
Comment (srmukher) [reply change/@fluentui-react-charting-5d7a9b00-f0eb-45e8-b014-adc12ef4a089.json]: done

**Review by AtishayMsft — APPROVED** [review]
